### PR TITLE
Release Notes for 3.5 RC

### PR DESCRIPTION
**Pull request by harikmenon**
Added notes

### CODE REVIEW

**Comment by emgarten** [comment]
https://github.com/NuGet/Home/issues/2956 was a major perf fix and is marked as 3.5 RC, I dont see it in the list here.

**Comment by harikmenon** [comment]
@emgarten fixed.

**Comment by emgarten** [comment]
https://github.com/NuGet/Home/issues/3150 was another major perf fix for project.json, for portable projects time went from 13s to restore to 5s

File: NuGet.Docs/Release-Notes/3.5-RC.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+#3.5 RC Release Notes
+
+[Full Changelog]()
```
Comment (yishaigalatzer) [inline NuGet.Docs/Release-Notes/3.5-RC.md]: Suggestion: Add a short summary: something like in this release we focused on bug fixes and added a 4 features.

File: NuGet.Docs/Release-Notes/3.5-RC.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+**Bugs:**
+
+* Install/restore of a package fails with "Package contains multiple nuspec files." - [#3231](https://github.com/NuGet/Home/issues/3231)
+
+* nuget pack forcefully adds .tt files to content folder no matter what - [#3203](https://github.com/NuGet/Home/issues/3203)
+
+* nuget pack csproj (with project.json) crashes if there are no packOptions and owner in json file - [#3180](https://github.com/NuGet/Home/issues/3180)
+
+* Performance: Improve ContentModel target framework parsing - [#3162](https://github.com/NuGet/Home/issues/3162)
```
Comment (yishaigalatzer) [inline NuGet.Docs/Release-Notes/3.5-RC.md]: We should breakout performance updates to a separate section, and provide sample improvment numbers

**Comment by emgarten** [comment]
https://github.com/NuGet/Home/issues/2807  is a notable change with how nuget handles .symbols.nupkg

File: NuGet.Docs/Release-Notes/3.5-RC.md (hunk 2 above)
Comment (emgarten) [inline NuGet.Docs/Release-Notes/3.5-RC.md]: That would be great, we should get some numbers for 3.4.4 vs 3.5.0-rc1

**Comment by emgarten** [comment]
**Features:**

SemVer 2.0.0 support in NuGet.Tools VSIX

**Comment by emgarten** [comment]
https://github.com/NuGet/Home/issues/2817

**Comment by yishaigalatzer** [comment]
@emgarten the semver 2.0 bug seems to be still open, and we need a large section explaining exactly what is now supported with samples and what is not supported

**Comment by emgarten** [comment]
@yishaigalatzer https://github.com/NuGet/Home/wiki/SemVer-2.0.0-support

**Comment by harikmenon** [comment]
@rrelyea : Can you send me the mail around the perf numbers I can incorporate into the release notes? Once I have that. I should be able to complete this.

File: NuGet.Docs/Release-Notes/3.5-RC.md (hunk 1 above)
Comment (harikmenon) [inline NuGet.Docs/Release-Notes/3.5-RC.md]: Fixed

**Comment by harikmenon** [comment]
@emgarten incorporated everything except SemVer support as discussed.

**Comment by harikmenon** [comment]
@yishaigalatzer @emgarten @rrelyea we are ready to go for release notes, incorporated all comments.

**Pull request merged**